terraform/cadet/vpc.tf

```diff
@@ -7,10 +7,32 @@ resource "aws_vpc" "cadet" {
   }
 }
 
+resource "aws_subnet" "private_a" {
```

Comment:
I think there is a requirement for the subnets to have different AZs, otherwise high availability is impossible